.github/workflows/workflow.yml

```diff
@@ -21,13 +21,13 @@ jobs:
         - name: Set up Python
           uses: actions/setup-python@master
           with:
-            python-version: 3.6
+            python-version: 3.9
 
         - name: Install dependencies
           run: |
             python -m pip install --upgrade pip
-            pip install -r requirements.txt
```

Comment:
why does this need to be removed? it seems like the tests are passing (and I can see in the checks that the dependencies do get installed) so it obviously isn't necessary, just curious.